engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-errata-modal.html

```diff
@@ -5,7 +5,6 @@ <h4 data-block="modal-header">Content Host Errata Management</h4>
     <form id="errataActionForm" name="errataActionForm" class="form" method="post" action="/katello/remote_execution">
       <input type="hidden" name="name" ng-value="errataActionFormValues.errata"/>
       <input type="hidden" name="remote_action" ng-value="errataActionFormValues.remoteAction"/>
-      <input type="hidden" name="scoped_search" ng-value="errataActionFormValues.search"/>
```

Comment:
I don't think we need the below line in `content-hosts-bulk-errata-modal.controller.js` since we removed the form input for search.

https://github.com/Katello/katello/blob/4a99b51048fc0816c3b4237f91448b39ae9af388/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-errata-modal.controller.js#L54

Reply:
I think you're right. The field was called 'scoped_search' anyway - so this was not working from what I can see